JsonSchema/JsonSchemaCacheWarmer.php

```diff
@@ -25,13 +28,16 @@ public function __construct(CachedSchemaStorage $cachedSchemaStorage)
         $this->cachedSchemaStorage = $cachedSchemaStorage;
     }
 
+    /**
+     * @inheritdoc
```

Comment:
inheritdoc has to be enclosed in `{@inheritdoc}` see https://www.phpdoc.org/docs/latest/guides/inheritance.html